### PR TITLE
added tabs,card var in Authentication

### PR DESCRIPTION
**Pull request by Sukumar-K7**

### CODE REVIEW

File: schemas/Authentication/index.ts [hunk 1]
```diff
@@ -39,6 +39,10 @@ export const Authentication: FeatureSchemaDefinition = {
         title: "[Authentication] Action Labels",
         value: "authentication.card.action-labels",
       },
+      {
+        title: "[Authentication] Center Aligned content",
```
[the remark this answers is not in the document]
Comment (NarsingRao13) [reply schemas/Authentication/index.ts:43]: Remove this and try to reuse the existing variant.

File: schemas/gift-cards/index.ts [hunk 2]
```diff
@@ -11,5 +11,23 @@ export const giftCards: FeatureSchemaDefinition = {
       value: "giftCards.manage-gift-card",
     },
   ],
-  variants: {},
+  variants: {
+    tabs: [
+      {
+        title: "[Gift Card] Manage Card Multiple Tabs",
```
Comment (Sukumar-K7) [inline schemas/gift-cards/index.ts:17]: image

File: schemas/gift-cards/index.ts [hunk 3]
```diff
@@ -11,5 +11,23 @@ export const giftCards: FeatureSchemaDefinition = {
...
+      },
+    ],
+    card: [
+      {
+        title: "[Gift Card] Check Balance Form",
+        value: "giftCards.card.check-balance-form",
+      },
+      {
+        title: "[Gift Card] Reload Card Form",
+        value: "giftCards.card.reload-card-form",
```
Comment (Sukumar-K7) [inline schemas/gift-cards/index.ts:28]: image

File: schemas/gift-cards/index.ts [hunk 4]
```diff
@@ -11,5 +11,23 @@ export const giftCards: FeatureSchemaDefinition = {
...
+    tabs: [
+      {
+        title: "[Gift Card] Manage Card Multiple Tabs",
+        value: "giftCards.tabs.manage-card-multiple-tabs",
+      },
+    ],
+    card: [
+      {
+        title: "[Gift Card] Check Balance Form",
+        value: "giftCards.card.check-balance-form",
```
Comment (Sukumar-K7) [inline schemas/gift-cards/index.ts:24]: image

**Review by Sukumar-K7 — COMMENTED** [review]
updated the changes...

**Pull request merged**